### DISCUSSION TITLE
Apply completion hotkey

### DISCUSSION THREAD

**Discussion by Jawell** [community/community#15258] [Copilot Conversations] [answered]
I use Copilot in WebStorm and I want to change hotkey to apply completion. For now, this hotkey is Tab. Often I just want to make Tab space in my code, but I can't do it because Tab apply completion from Copilot. And each time I need to press ESC. But if I need to make several Tab spaces, I need each time press Esc
![Untitled](https://user-images.githubusercontent.com/21054523/164205180-fd3811cb-a222-4cd2-9d7a-a5df209d1111.gif)

**Comment by jansorg** [comment, marked as answer]
You can change the hotkey at `Preferences > Keymap > Plugins > GitHub Copilot`.

**Reply by zayadur** [reply]
I'm on a vanilla install of VS Code with GitHub Copilot. There is no Keymap option under Preferences. Is there another way to access Copilot hotkeys?
